sklearn/tests/test_config.py

```diff
@@ -3,36 +3,37 @@
 
 
 def test_config_context():
-    assert_equal(get_config(), {'assume_finite': False})
+    assert_equal(get_config(),
```

Comment:
Just curious, why did you not user `get_config()['assume_finite']` here?

Since you are touching this code, bonus points if you use bare asserts rather than `assert_equal`.

Reply:
Because we want to check *all* defaults.
I'll see about winning those bonus points! Thanks